Cargo.toml

```diff
@@ -15,15 +15,15 @@ categories = [ "cryptography::cryptocurrencies" ]
 normal = ["async-trait"]
 
 [dependencies]
-bee-rest-api = { git = "https://github.com/iotaledger/bee", rev = "e9ef4d6725a9b63756107618c60b75cf1aca56d1", default-features = false }
-bee-message = { git = "https://github.com/iotaledger/bee", rev = "e9ef4d6725a9b63756107618c60b75cf1aca56d1", default-features = false, features = [ "serde1", "dto" ] }
-bee-pow = { git = "https://github.com/iotaledger/bee", rev = "e9ef4d6725a9b63756107618c60b75cf1aca56d1", default-features = false }
+bee-rest-api = { git = "https://github.com/iotaledger/bee", rev = "4b5b74e5b7aedd0f52313af8814673d8a1f571bd", default-features = false }
+bee-message = { git = "https://github.com/iotaledger/bee", rev = "4b5b74e5b7aedd0f52313af8814673d8a1f571bd", default-features = false, features = [ "serde", "dto" ] }
+bee-pow = { git = "https://github.com/iotaledger/bee", rev = "4b5b74e5b7aedd0f52313af8814673d8a1f571bd", default-features = false }
 # bee-rest-api = { path = "../bee/bee-api/bee-rest-api",  default-features = false }
-# bee-message = { path = "../bee/bee-message",  default-features = false, features = [ "serde1" ] }
+# bee-message = { path = "../bee/bee-message",  default-features = false, features = [ "serde" ] }
```

Comment:
why is this dependency not used? can this be removed?

Reply:
I left it there because I often test new changes with bee locally, but now not so often anymore and if you guys don't use it we can also remove it

Reply:
okay if you don't need it so much anymore then i guess it's better to remove it